### PR TITLE
PLT-2257 Reorganized system console

### PR DESCRIPTION
**Pull request by hmhealey**

### CODE REVIEW

File: webapp/components/admin_console/brand_image_setting.jsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Client from 'utils/web_client.jsx';
+import * as Utils from 'utils/utils.jsx';
+
+import FormError from 'components/form_error.jsx';
+import {FormattedHTMLMessage, FormattedMessage} from 'react-intl';
+
+export default class GeneratedSetting extends React.Component {
```
Comment (hmhealey) [inline webapp/components/admin_console/brand_image_setting.jsx]: Fix name

**Comment by hmhealey** [comment]
Spinmint set up at http://harrison.spinmint.com:8065/ with test user `test@test.com` using password `passwd`

**Comment by hmhealey** [comment]
Marking this as DO NOT MERGE since we most likely won't take it until after the next release

**Comment by it33** [comment]
Adding test server to try this out,

**Comment by hmhealey** [comment]
Closing this since it'll be resubmitted against master after we branch for the next release

**Pull request closed without merging**